flexget/plugins/input/trakt_emit.py

```diff
@@ -59,16 +59,16 @@ def on_task_input(self, task, config):
                 log.warning('The list "%s" is empty.' % config['list'])
                 return
             for item in data:
-                if item['show'] is not None:
-                    if not item['show']['title']:
-                        # Seems we can get entries with a blank show title sometimes
-                        log.warning('Found trakt list show with no series name.')
-                        continue
-                    trakt_id = item['show']['ids']['trakt']
-                    listed_series[trakt_id] = {
-                        'series_name': item['show']['title'],
-                        'trakt_id': trakt_id,
-                        'tvdb_id': item['show']['ids']['tvdb']}
+                    if getattr(item, 'show'):
+                        if not item['show']['title']:
+                            # Seems we can get entries with a blank show title sometimes
+                            log.warning('Found trakt list show with no series name.')
+                            continue
+                        trakt_id = item['show']['ids']['trakt']
+                        listed_series[trakt_id] = {
+                            'series_name': item['show']['title'],
+                            'trakt_id': trakt_id,
+                            'tvdb_id': item['show']['ids']['tvdb']}
```

Comment:
Looks like your whole block got indented too far. Also, this doesn't seem right, this will get an attribute rather than the value for the dictionary key. I think `if item.get('show'):` is what you were looking for.